dat_utils/__init__.py

```diff
@@ -1,11 +1,4 @@
-"""
-    .___       __                    __  .__.__          
-  __| _/____ _/  |_           __ ___/  |_|__|  |   ______
- / __ |\__  \\   __\  ______ |  |  \   __\  |  |  /  ___/
-/ /_/ | / __ \|  |   /_____/ |  |  /|  | |  |  |__\___ \ 
-\____ |(____  /__|           |____/ |__| |__|____/____  >
-     \/     \/                                        \/ 
-"""
+""" dat-utils """
```

Comment:
:cry:

Reply:
yeah I know, if anyone know how to keep it, I'm fine. The reason I removed it was I was getting the following warning:

```bash
.venv/src/django-astrosat-users/astrosat_users/__init__.py:14
  /home/app/.venv/src/django-astrosat-users/astrosat_users/__init__.py:14: DeprecationWarning: invalid escape sequence \ 
    """
```

Reply:
You can use the `r` prefix to ignore escape sequences, so:
```
r"""
...
"""
```

https://docs.python.org/3.7/reference/lexical_analysis.html#string-and-bytes-literals

Reply:
I did that with the others as you can see. I thought this was only for regex's, but happy to use it here, if it works. My problem is, I have no way of proving it in this library, testing has not been setup. I didn't want to go to that level for this, but testing should be added at some point.

I'll add the `r` here as well, but we won't know if it works or not until we merge and use it.

Reply:
I've re-introduced the ascii art and prefixed it with `r` to treat it as a **raw string**. We will need to add tests at some point to prove we are not introducing any such problems in the future, but that can be done later.